serverless.yml

```diff
@@ -29,6 +29,35 @@ stepFunctions:
             Type: Task
             Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-retriever-capture-raw-load-${self:provider.stage}-iowRetrieverCapture
             TimeoutSeconds: 60
+            Retry:
+              - ErrorEquals:
+                - States.TaskFailed
+                IntervalSeconds: 3
+                MaxAttempts: 3
+                BackoffRate: 2
+            Next: tsTypeRouterTask
+          tsTypeRouterTask:
+            Type: Task
+            Resource: arn:aws:lambda:${self:provider.region}:${self:custom.accountNumber}:function:aqts-ts-type-router-${self:provider.stage}-determineRoute
+            TimeoutSeconds: 60
+            Retry:
+              - ErrorEquals:
+                - States.TaskFailed
+                IntervalSeconds: 3
+                MaxAttempts: 3
+                BackoffRate: 2
+            Next: tsTypeRouterChoice
+          tsTypeRouterChoice:
+            Type: Choice
+            Choices:
+              - Variable: "$.type"
+                StringEquals: tsDescription
+                Next: tsDescription
```

Comment:
It might vomit because `tsDescription` doesn't exist as a step yet. Might want to add a pass step until it's ready.